internal/weldr/api.go

```diff
@@ -2347,12 +2347,21 @@ func (api *API) composeHandler(writer http.ResponseWriter, request *http.Request
 		return
 	}
 
+	var packages []rpmmd.PackageSpec
+	// TODO: introduce a way to query these from the manifest / image type
+	// BUG: installer/container image types will have empty package sets
```

Comment:
🐛

Reply:
Yeah, this bug remains unchanged, I just thought I would point it out.